app/MMB/src/scenes/tripServices/insurance/insuranceOverviewScene/orderSummary/OrderSummary.js

```diff
@@ -2,45 +2,48 @@
 
 import * as React from 'react';
 import { View } from 'react-native';
-import {
-  StyleSheet,
-  Color,
-  Text as NativeText,
-  Price,
-} from '@kiwicom/mobile-shared';
+import { StyleSheet, Color, Text, Price } from '@kiwicom/mobile-shared';
 import { SeparatorFullWidth } from '@kiwicom/mobile-navigation';
 import { Translation } from '@kiwicom/mobile-localization';
 
-function Text({ children }) {
-  return <NativeText style={styleSheet.text}>{children}</NativeText>;
-}
+import { withInsuranceContext } from '../InsuranceOverviewSceneContext';
+import InsuranceRow from './InsuranceRow';
+
+type Props = {|
+  +amount: number,
+  +currency: string,
+|};
 
-export default function OrderSummary() {
+function OrderSummary(props: Props) {
   return (
     <View style={styleSheet.wrapper}>
-      <View style={styleSheet.row}>
-        <View style={styleSheet.item}>
-          <Text>
-            <Translation passThrough="TODO" />
-          </Text>
-        </View>
-        <Price amount={-1} currency="WTF" style={styleSheet.price} />
-      </View>
+      <InsuranceRow insuranceType="TRAVEL_PLUS" />
```

Comment:
It is just the translation that is different right?

I think it is better to pass in the translation than the insurance type in that case.

Reply:
Oh, right, there are something more

Reply:
The issue is that I compute the quantity there and I need it in the translation.